### PR TITLE
Feature/news reader

### PR DESCRIPTION
**Pull request by iamandiianamariaa**

### CODE REVIEW

File: app/src/main/java/com/iamandiiana/newsreader/DemoApplication.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.iamandiiana.newsreader;
+
+import android.app.Application;
+
+import com.iamandiiana.data.di.RepoModule;
+
+public class DemoApplication extends Application {
```
Comment (Misca) [inline app/src/main/java/com/iamandiiana/newsreader/DemoApplication.java]: DemoApplication seems a bit too generic, let's rename this class to something that makes it look like part of out app, like "NewsApplication"

File: app/src/main/java/com/iamandiiana/newsreader/feature/newslist/MainActivity.java [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
 import android.os.Bundle;
 
-import com.iamandiiana.newsreader.ui.main.MainFragment;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.iamandiiana.newsreader.R;
+import com.iamandiiana.newsreader.feature.newslist.fragment.NewsListFragment;
 
 public class MainActivity extends AppCompatActivity {
```
Comment (Misca) [inline app/src/main/java/com/iamandiiana/newsreader/feature/newslist/MainActivity.java]: Let's also rename this activity to something more specific

**Review by Misca — CHANGES_REQUESTED** [review]
Almost there, good job.

File: app/src/main/java/com/iamandiiana/newsreader/DemoApplication.java (hunk 1 above)
Comment (iamandiianamariaa) [reply app/src/main/java/com/iamandiiana/newsreader/DemoApplication.java]: Done

File: app/src/main/java/com/iamandiiana/newsreader/feature/newslist/MainActivity.java (hunk 2 above)
Comment (iamandiianamariaa) [reply app/src/main/java/com/iamandiiana/newsreader/feature/newslist/MainActivity.java]: Done

**Review by Misca — APPROVED** [review]
Good job, Ana!  💯

**Pull request merged**